### ISSUE TITLE
Support golang (.go) files

### ISSUE THREAD

**Issue by liujinf** [ejwa/gitinspector#86]
support golang's comment and metrics.

**Comment by adam-waldenberg** [comment]
While implementing comments obviously is trivial, the implementation of metrics would have to wait a while, as I am not familiar enough with the Go syntax and I would most likely miss something :).

However, pull requests are always welcome. The metrics system for cyclomatic complexity is very simple and just makes use of a list of entry and exit tokens defined as regular expressions. A short explanation on how to add language support is mentioned in issue #59.

**Comment by adam-waldenberg** [comment]
Added with #166 - will be merged in 0.5.0.

**Comment by MB175** [comment]
Is this still planed ? would love the go support

**Comment by adam-waldenberg** [comment]
@MB175 Yes. In the meantime just apply the pull request. Not much time for gitinspector lately, but one of these days I will take a week or two to get things up to scratch.

**Comment by adam-waldenberg** [comment]
Go support has now landed on master with commit c997472, based on the work in #166.

Go files are included in the analysis by default and get the estimated lines of code and cyclomatic complexity metrics. Comment handling for Go was already in place.
